tools/benchmark_tool/openvino/tools/benchmark/parameters.py

```diff
@@ -163,6 +163,8 @@ def parse_args():
                            "counters and latency for each executed infer request.")
     stat.add_argument('-report_folder', '--report_folder', type=str, required=False, default='',
                       help="Optional. Path to a folder where statistics report is stored.")
+    args.add_argument('-json_stats', '--json_stats', type=str2bool, required=False, default=False, nargs='?', const=True,
+                      help="Optional. Enables JSON-based statistics output (by default reporting system will use CSV format). Should be used together with -report_folder option.")
```

Comment:
Running `../bin/intel64/Release/benchmark_app -niter 2 -m ~/.cache/omz/intel/face-detection-0200/FP16/face-detection-0200.xml -report_folder . -report_type detailed_counters -json_stats` saves `benchmark_detailed_counters_report.json` in which `detailed_performance` is a list of two big elements. Running `py ../tools/benchmark_tool/benchmark_app.py -niter 2 -m ~/.cache/omz/intel/face-detection-0200/FP16/face-detection-0200.xml -report_folder . -report_type detailed_counters -json_stats` saves only one big element into `detailed_performance` entry in `benchmark_detailed_counters_report.json`
Please align the behavior.

Reply:
Aligned!